recipe/meta.yaml

```diff
@@ -1,18 +1,21 @@
 {% set name = "pyinstaller-hooks-contrib" %}
-{% set version = "2022.14" %}
+{% set version = "2024.7" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/pyinstaller/pyinstaller-hooks-contrib/archive/v{{ version }}.tar.gz
-  sha256: e5c94b9720545f8dadc105270ae16cafb2b4189e08dbf469104218cdcda2abef
+  sha256: 6bc703891fe21e8b983483a87b1a21a2ec6d3329575a6955457e639449efe78b
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-build-isolation --no-deps
   skip: True  # [py<37]
+  entry_points:
+    - hook-dirs = _pyinstaller_hooks_contrib.hooks:get_hook_dirs
+    - tests = _pyinstaller_hooks_contrib.tests:get_test_dirs
```

Comment:
```suggestion
```
It's only for console_scripts. See https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html